opsdroid/logging.py

```diff
@@ -84,12 +84,15 @@ def configure_logging(config):
     rootlogger.setLevel(log_level)
 
     formatter = logging.Formatter("%(levelname)s %(name)s: %(message)s")
+    with contextlib.suppress(KeyError):
+        if config["logging"]["timestamp"]:
+            formatter = logging.Formatter("%(asctime)s %(levelname)s %(name)s: %(message)s")
 
     with contextlib.suppress(KeyError):
         if config["logging"]["extended"]:
-            formatter = logging.Formatter(
-                "%(levelname)s %(name)s.%(funcName)s(): %(message)s"
-            )
+            formatter = logging.Formatter("%(levelname)s %(name)s.%(funcName)s(): %(message)s")
+            if config["logging"]["timestamp"]:
+                formatter = logging.Formatter("%(asctime)s %(levelname)s %(name)s: %(message)s")
```

Comment:
This has all become a bit messy. It would be nice to build our format string in pieces and then create one `logging.Formatter` object.

Could I ask you to refactor this?

Reply:
for sure, I was thinking the same, but wasn't sure if it would be your preference. will work on something